src/strings.cpp

```diff
@@ -281,6 +289,20 @@ char *GetString(char *buffr, StringID string, const char *last)
 	return GetStringWithArgs(buffr, string, &_global_string_params, last);
 }
 
+/**
+ * Migration helper to get a parsed string with most special stringcodes replaced by the string parameters.
```

Comment:
I would leave out "Migration helper" .. who knows, maybe you get bored tomorrow and never continue. Which is fine. But then this comment makes little sense :D